packages/react-devtools-shared/src/backend/fiber/renderer.js

```diff
@@ -1102,6 +1102,7 @@ export function attach(
 
   function getComponentStack(
     topFrame: Error,
+    // TODO: is this needed now that owner stacks have landed? e.g. for backwards compat of devtools?
```

Comment:
@sebmarkbage same here - can I remove the `enableOwnerStacks` value returned?

Reply:
No, because it feature detects the version.

Reply:
Yeah, all changes in React DevTools files should be compatible with older versions of React and its internals.